src/concat_bar_layer_circle.vl.js

```diff
@@ -1,75 +1,35 @@
 import * as vl from 'vega-lite-api';
```

Comment:
This PR shouldn't have any changes to this file. It should just include changes to the embedded_csv example. Otherwise it's hard to understand how this PR compares to the other one.